packages/bot-skeleton/src/scratch/blocks/Math/math_number.js

```diff
@@ -35,7 +35,7 @@ Blockly.Blocks.math_number = {
         };
     },
     numberValidator(input) {
-        if (/^-?([0][,.]|[1-9]+[,.])?([0]|[1-9])*$/.test(input)) {
+        if (/^-?([0][.]|[1-9]+[.])?([0]|[1-9])*$/.test(input)) {
```

Comment:
comma is not allowed anymore?

Reply:
No, we only want to accept dots as fractional separators to be consistent with DTrader and make it less confusing.